### PR TITLE
KOGITO-7775: Add integration test for json syntax highlighting for SWF editor in VS Code

### PR DESCRIPTION
**Pull request by barboras7**

### CODE REVIEW

File: packages/vscode-extension-serverless-workflow-editor/it-tests/serverless-workflow-editor-extension-syntax-highlight.test.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      uniqueColors.length
+    );
+  });
+
+  async function getWebElementByDisplayedText(displayedText: string): Promise<WebElement> {
+    return await driver.findElement(By.xpath("//*[text()[contains(.,'" + displayedText + "')]]"));
+  }
+
+  async function resolveWebElementColor(webElement: WebElement): Promise<string> {
+    return await (await Promise.resolve(webElement)).getAttribute("class");
```
Comment (tomasdavidorg) [inline packages/vscode-extension-serverless-workflow-editor/it-tests/serverless-workflow-editor-extension-syntax-highlight.test.ts]: I would say this is not necessary.

```suggestion
    return await webElement.getAttribute("class");
```

File: packages/vscode-extension-serverless-workflow-editor/it-tests/serverless-workflow-editor-extension-syntax-highlight.test.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    const stringValue2WebElement = await getWebElementByDisplayedText("HelloWorldWorkflow");
+    const stringValue1Color = await resolveWebElementColor(stringValue1WebElement);
+    const stringValue2Color = await resolveWebElementColor(stringValue2WebElement);
+
+    // get color code of json boolean value
+    const booleanValueWebElement = await getWebElementByDisplayedText("true");
+    const booleanValueColor = await resolveWebElementColor(booleanValueWebElement);
+
+    // check that properties have the same color
+    expect(property1Color == property2Color, "All properties should be the same color").to.be.true;
```
Comment (tomasdavidorg) [inline packages/vscode-extension-serverless-workflow-editor/it-tests/serverless-workflow-editor-extension-syntax-highlight.test.ts]: Could you please use 
```suggestion
    expect(property1Color, "All properties should be the same color").equal(property2Color);
```

**Review by tomasdavidorg — CHANGES_REQUESTED** [review]
Please take a look to my comments + please remove the json svg.

**Review by tomasdavidorg — APPROVED** [review]
Thanks 🐼

**Review by tiagobento — APPROVED** [review]

**Pull request merged**